src/Backend/Modules/ContentBlocks/Actions/ContentBlockAdd.php

```diff
@@ -12,26 +12,22 @@
 use Backend\Core\Engine\Authentication;
 use Backend\Core\Engine\Base\ActionAdd as BackendBaseActionAdd;
 use Backend\Core\Engine\Model as BackendModel;
-use Backend\Modules\ContentBlocks\Command\CreateContentBlock;
-use Backend\Modules\ContentBlocks\Event\ContentBlockCreated;
-use Backend\Modules\ContentBlocks\Form\ContentBlockType;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Command\CreateContentBlock;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Event\ContentBlockCreated;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockType;
+use Symfony\Component\Form\Form;
 
 /**
  * This is the add-action, it will display a form to create a new item
  */
-class Add extends BackendBaseActionAdd
+class ContentBlockAdd extends BackendBaseActionAdd
 {
     public function execute(): void
     {
         parent::execute();
 
-        $form = $this->createForm(
-            ContentBlockType::class,
-            null,
-            ['theme' => $this->get('fork.settings')->get('Core', 'theme', 'Core')]
-        );
-
-        $form->handleRequest($this->get('request'));
+        /** @var Form $form */
+        $form = $this->getForm();
 
         if (!$form->isValid()) {
```

Comment:
checking if a form is valid before checking if a form is submitted is deprecated in symfony 3.3 and removed in symfony 4, since we will need to upgrade for fork 5 this will need to be changed here as well